azure_user_defined_tags.md

```diff
@@ -0,0 +1,459 @@
+---
+title: apply-user-defined-tags-to-all-azure-resources-created-by-openShift
```

Comment:
Please pick a domain and move this file into a subdirectory. There's an existing enhancement for custom tags on AWS, so this could go in the same directory as that one, or it could go into a directory like `cloud-integration`.

Reply:
I was unsure where to place this. Thank you for clearing up, will add it to enhancements/api-review